jobbergate-api/jobbergate_api/apps/permissions.py

```diff
@@ -16,3 +21,22 @@ class Permissions(str, Enum):
     JOB_SCRIPTS_EDIT = "jobbergate:job-scripts:edit"
     JOB_SUBMISSIONS_VIEW = "jobbergate:job-submissions:view"
     JOB_SUBMISSIONS_EDIT = "jobbergate:job-submissions:edit"
+
+
+def check_owner(
+    owner_email: Union[str, EmailStr, None],
+    requester_email: Union[str, EmailStr, None],
+    entity_id: int,
+    entity_name: str,
+):
+    """
+    Assert ownership of an entity and raise a 403 exception with message on failure.
+    """
+    if not requester_email == owner_email:
```

Comment:
Small suggestion

```suggestion
    if requester_email != owner_email:
```

Reply:
Lol, good catch. I was trying to hustle through and, yeah...

Addressed in commit e6893b5